roles/apache/tasks/backup.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Ansible check /etc/httpd exists
+  stat:
+    path: /etc/httpd
+  register: httpd_folder
+
+- name: create backup directory
+  file:
+    path: "{{ backup_dir }}/httpd_{{ansible_date_time.iso8601_basic_short}}"
+    state: directory
+    mode: 0755
+  when: httpd_folder.stat.exists and httpd_folder.stat.isdir
+
+- name: copy httpd folder to backup directory
+  copy:
```

Comment:
Instead of simple copy, we can also create a tarball. 
https://docs.ansible.com/ansible/2.9/modules/archive_module.html